1_fetch/src/get_sb_data.R

```diff
@@ -131,10 +131,17 @@ prep_feature_vars <- function(sb_var_data, sites, retain_vars) {
   #'@return data frame with COMID column appended by all feature variables of interest; 
   #'time-varying features converted to long-term averages where applicable
   
+  options(dplyr.summarise.inform = FALSE)
+  
   data <- sites %>%
     select(COMID, all_of(retain_vars)) %>%
     mutate(across(where(is.character), as.numeric))
```

Comment:
I think this will work. Then the mutate in the if statement for GAGES_ID is not needed
```suggestion
    mutate(across(where(is.character)  & !starts_with('ID'), as.numeric))
```